<ui version="4.0" >
 <class>BasicDynamicColoringProgramEditor</class>
 <widget class="QWidget" name="BasicDynamicColoringProgramEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>166</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Mixer</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QCheckBox" name="mixerEnabled" >
         <property name="text" >
          <string>enabled</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="labelMixerJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Jitter:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="KIntNumInput" name="mixerJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="suffix" >
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="labelMixerSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Sensor:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="KisSensorSelector" name="mixerSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Hue</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QCheckBox" name="hueEnabled" >
         <property name="text" >
          <string>enabled</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="labelHueJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Jitter:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="KIntNumInput" name="hueJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="suffix" >
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="labelHueSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Sensor:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="KisSensorSelector" name="hueSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Saturation</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QCheckBox" name="saturationEnabled" >
         <property name="text" >
          <string>enabled</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="labelSaturationJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Jitter:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="KIntNumInput" name="saturationJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="suffix" >
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="labelSaturationSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Sensor:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="KisSensorSelector" name="saturationSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Brightness</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QCheckBox" name="brightnessEnabled" >
         <property name="text" >
          <string>enabled</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="labelBrightnessJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Jitter:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="KIntNumInput" name="brightnessJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="suffix" >
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="labelBrightnessSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Sensor:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="KisSensorSelector" name="brightnessSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KisSensorSelector</class>
   <extends>QComboBox</extends>
   <header location="global" >kis_sensor_selector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
